src/ActionsMenuItem.js

```diff
@@ -54,14 +54,14 @@ export class ActionsMenuItem extends React.Component< ActionsMenuItemProps, {} >
             icon = React.createElement(iconType);
         }
 
-        // Disable the checkbox if the button is allowed and used.
-        const isCheckboxDisabled = (this.props.isAllowed && this.props.isUsed);
+        // Disable the checkbox if the button is disallowed or used.
```

Comment:
This comment is not accurate. It was accurate as it was: "Disable the checkbox if the button is allowed and used."